### DISCUSSION TITLE
Godot

### DISCUSSION THREAD

**Discussion by MustaphaRashiduddin** [extism/extism#188] [General]
i'm just asking how this would be integrated into godot (gdscript) would they have to include this using c++ (the main langauge they use to make godot) and make bindings for that in gdscript? because gdscript is not a standalone language.

**Comment by nilslice** [comment]
@MustaphaRashiduddin - I'm not very familiar with Godot engine, but it appears that it is possible to link to native shared libraries from GDScript using GDNative: https://docs.godotengine.org/en/stable/tutorials/scripting/gdnative/what_is_gdnative.html

Extism is a shared library, and it may even be possible to use the C/C++ SDKs we maintain and bridge them to GDScript.
